src/components/dashboard/DashboardDetail.js

```diff
@@ -115,14 +115,14 @@ const mapDispatchToProps = {
 }
 
 // make sure that an action is an object, for UI purpose
-const postprocessSerdeStruct = (action) => typeof action == 'object' ? [action] : [{[action]: {}}]
+const postprocessSerdeStruct = (action) => typeof action == 'object' ? [{[action.kind]: [action.args]}] : [{[action]: {}}]
 
 const mapStateToProps = ({ account }) => {
     const transactions = account.transactions 
-        ? account.transactions.flatMap(t => t.actions.map((a) => ({
+        ? account.transactions.map(t => ({
```

Comment:
@vgrichina Interesting, so we don't care if those actions were a part of a single transaction; is this intended?

Reply:
@frol any reason for us to care? It's displayed as a flat list anyway. Batch transaction concept as we have now generally isn't very useful, as it only works for same receiver (while in apps it's most interesting to batch calls to different receivers).